### ISSUE TITLE
Accessibility settings

### ISSUE THREAD

**Issue by thewizz69** [Angel-Studio/SoundTap#15]
On Samsung galaxy fold 4 even after enabling all settings permissions, you still can't progress past the accessibility service intro setring.

**Comment by JulesPvx** [comment]
Thanks for the feedback, I'll try to take a look at it

**Comment by JulesPvx** [comment]
Did you installed it using the apk ? If that's the case could you try with the play store version and let me know if it works that way

**Comment by thewizz69** [comment]
Hi,

I tried both ways.

________________________________

**Comment by JulesPvx** [comment]
Well, I'll check that then x) don't know what is causing it at all

**Comment by Keatondickens** [comment]
Yeah I've got the same issue with the accessibility issue sadly I've tried a few things but nothing worked

**Comment by JulesPvx** [comment]
Samsung too ?

**Comment by Keatondickens** [comment]
> Samsung too ?

Na I'm on a pixel 6 pro I don't know if it's a different app causing issues but I've never had this in any other app.

**Comment by JulesPvx** [comment]
Okay thanks, I'll add a button to continue without the need to enable it then.

**Comment by austinevick** [comment]
I am facing the same issue on my infinix phone.

**Comment by guguss-31** [comment]
I have the same problem on my samsung phone. (I tried with play store and release apk) 

Galaxy A20e (SM-A202F/DS)
OneUI 3.1, Android 11
